source/vistas/cli/startup_project.py

```diff
@@ -0,0 +1,15 @@
+import click
+from vistas.cli import cli
+from vistas.ui.app import App
+import os
+
+
+@cli.command(short_help='Start VISTAS with a predefined project loaded.')
+@click.argument('project', type=click.Path(exists=True))
+def startup_project(project):
```

Comment:
Since this is an action, I think a name like `load_project` would be more appropriate. `startup_project` feels more like an argument name.